paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/IdealBankConfig.kt

```diff
@@ -0,0 +1,51 @@
+package com.stripe.android.paymentsheet.elements;
+
+import com.stripe.android.paymentsheet.elements.common.DropdownConfigInterface
+
+internal class IdealBankConfig : DropdownConfigInterface {
+    override val debugLabel = "idealBank"
+    override val label = com.stripe.android.R.string.title_bank_account
+
+    override fun convertToDisplay(paramFormatted: String?) =
+        PARAM_TO_DISPLAY[paramFormatted] ?: ""
+
+    override fun convertToPaymentMethodParam(displayFormatted: String) =
+        DISPLAY_TO_PARAM[displayFormatted]
+
+    override fun getItems(): List<String> = DISPLAY_TO_PARAM.keys.sortedBy { it }
+
+    companion object {
+        // TODO: Need to determine the correct way to pass junit default locale
+        val DISPLAY_TO_PARAM: Map<String, String> = mapOf(
+            "ABN AMRO" to "abn_amro",
+            "ASN Bank" to "asn_bank",
+            "Bunq" to "bunq",
+            "Handelsbanken" to "handelsbanken",
+            "ING" to "ing",
+            "Knab" to "knab",
+            "Moneyou" to "moneyou",
+            "Rabobank" to "rabobank",
+            "Revolut" to "revolut",
+            "RegioBank" to "regiobank",
+            "SNS Bank (De Volksbank)" to "sns_bank",
+            "Triodos Bank" to "triodos_bank",
+            "Van Lanschot" to "van_lanschot",
+        )
```

Comment:
You can do the following so that you don't need to maintain two lists.

```
val DISPLAY_TO_PARAM = PARAM_TO_DISPLAY.map { it.value to it.key }.toMap()
```

Reply:
Have refactored in a slightly different way